Module_08/ex02/README.md

```
@@ -0,0 +1,96 @@
+
+1. Creating and populating a `MutantStack`
+2. Traversing elements using regular iterators
+3. Performing standard stack operations (push, pop, top)
+4. Using constant iterators for read-only access
+5. Traversing in reverse order with reverse iterators
+6. Comparing functionality with `std::list` for reference
+
+## Helper Functions
+
+The implementation includes UI helper functions for better visualization:
+- `headerPrinter`: Prints a formatted header with text
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)